build/dockerfiles/test_entrypoint.sh

```diff
@@ -0,0 +1,350 @@
+#!/bin/bash
+#
+# Copyright (c) 2018-2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+METAS_DIR=$(mktemp -d)
+
+function cleanup() {
+    rm -rf  "${METAS_DIR}";
+}
+trap cleanup EXIT
+
+RED="\e[31m"
+GREEN="\e[32m"
+RESETSTYLE="\e[0m"
+BOLD="\e[1m"
+
+function initTest() {
+    echo -e "${BOLD}\n🏃 ${1}${RESETSTYLE}"
```

Comment:
emojis sometimes break stuff... is this needed?

Reply:
for test, visually it looks better when reading the logs in CI

Reply:
For instance: 
![Selection_347](https://user-images.githubusercontent.com/650571/116875844-dbffc900-ac1b-11eb-8b5e-7ae19dbc5eb7.png)

Reply:
:+1: